src/CacheInterface.php

```diff
@@ -148,22 +148,22 @@ public function getMultiple(array $keys, $default = null);
      * @param ?float $ttl    Optional. The TTL value of this item.
```

Comment:
I'm assuming we want to keep the param docblock because of the comment what it is for?

Reply:
Yes, I agree the docblock isn't necessarily required here, but I'd vote to keep this as is in case we want to add additional details such as types that can not be expressed in PHP (e.g. template types and array shapes). Same idea also applies to existing PSRs in v2 and Promise v3 APIs and it looks like this would be community consensus.

Reply:
I'd slightly vote differently, we always drop the dockblock if the type matches the native type hint, unless:
* We add a comment about the parameter
* We need to include template types/array shapes/class-string etc etc